packages/aws-cdk-lib/aws-applicationautoscaling/test/target-tracking.test.ts

```diff
@@ -91,6 +91,26 @@ describe('target tracking', () => {
     });
   });
 
+  test('test setup target tracking on predefined metric for SAGEMAKER_VARIANT_CONCURRENT_REQUESTS_PER_MODEL_HIGH_RESOLUTION', () => {
```

Comment:
can you check the other enum values as well. We need to have a good coverage to avoid deleting any of these values by mistake

Reply:
added other metrics too.